src/macrofier/engine.mjs

```diff
@@ -693,7 +694,12 @@ function sortSchemasByReference(schemas = []) {
   return schemas
 }
 
-const isEnum = x => x.type && x.type === 'string' && Array.isArray(x.enum) && x.title
+const getEnum = x => x.type && x.type === 'array' ? x.items : x
```

Comment:
why is this called `getEnum` if it's returning any array's items?

Reply:
it just returning array items alone, will add check for inside too

Reply:
done